### ISSUE TITLE
Discussion: Confusion about installing old versions of formula

### ISSUE THREAD

**Issue by j1wilmot** [Homebrew/brew#1838]
Firstly, thanks for homebrew! It's a great project.

I've been looking through documentation / stack overflow / google searches for information on how to install older versions of formula. Based on searches it seems like this is a fairly common use case scenario, but there doesn't seem to be a best practice on how to do this. From what I can gather homebrew/versions has been deprecated, so it's not clear to me how to best do this. I see http://docs.brew.sh/Versions.html talks about having multiple active versions, but doesn't mention legacy versions.

In short, is there a best practice way of installing older versions of applications? Is this documented somewhere that I missed? What is the projects' take on older application versions?  

Many thanks.

**Comment by MikeMcQuaid** [comment]
> In short, is there a best practice way of installing older versions of applications? Is this documented somewhere that I missed? What is the projects' take on older application versions?

This isn't something that's generically supported but just provided for certain applications. It's not completely ready yet so not completely documented yet. We'll likely announce it in the blog post for a later release.
